devel/release.md

```diff
@@ -32,6 +32,21 @@ git pull upstream master
 You may need to fix some conflicts. For auto-generated files, you can commit
 either version. They will be updated to the current version in the next step.
 
+A workaround for merge conflicts is to manually cherrypick the change from the
+master branch to the release branch. See https://github.com/kubernetes-client/python/pull/1411
+for examples.
```

Comment:
some clarification on the reasoning for this (i.e. parallel multiple releases), also there is a step about committing snapshot to master, that should be optional initially, and when all in-flight releases are completed, the snapshot should be updated to the latest release.